setup.py

```diff
@@ -44,7 +44,7 @@
 ]
 
 PACKAGE = {
-    'version': '3.0.6',
+    'version': '3.0.7',
```

Comment:
I think this warrants a minor bump, no?

Reply:
I don't know, does it?

It's actually just a bug fix. But fine, bumped in 64fbefc